network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/group_listing_block.html

```diff
@@ -0,0 +1,41 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block block_content %}
+  <div class="tw-row tw-my-4 large:tw-mb-7">
+    <div class="tw-px-4 tw-w-full">
+      <h2 class="tw-pt-2 tw-pb-3 tw-border-t tw-inline-block">{{ self.title }}</h2>
+    </div>
+
+    <div class="tw-space-y-4 large:tw-space-y-5">
+      {% for block in self.cards %}
+        {% with card=block %}
+          {% image card.image fill-690x388 as img %}
+            {% if card.url %}
+              <a href="{{ card.url }}" class="tw-block tw-group hover:tw-no-underline focus:tw-no-underline hover:tw-text-black focus:tw-text-black" aria-label="{{ card.title }}">
+            {% endif %}
+            <div class="tw-px-4 tw-grid tw-grid-cols-1 large:tw-grid-cols-3 tw-gap-4 large:tw-gap-6">
+              <div class="tw-col-span-1">
+                <img class="tw-w-full large:tw-w-auto large:tw-max-w-[345px]" src="{{ img.url }}" alt="{{ card.alt_text }}" width="{{ img.width }}" height="{{ img.height }}"/>
+              </div>
+              <div class="tw-col-span-1 large:tw-col-span-2 large:tw-items-center large:tw-flex">
+                <div class="tw-bg-white tw-relative tw-w-full tw-flex tw-flex-col">
+                  <span class="tw-h6-heading tw-text-gray-40 tw-py-1 tw-mb-0">{{ card.meta_data }}</span>
+                  <span class="tw-h3-heading tw-mb-2 tw-block
```

Comment:
Can yo add some context why we are using `span`s here? It looks like we want them to act like block rather than inline elements.

Reply:
WIth this being a h3 now we no longer need the block class so i've removed it now 😄  good catch